docs/reference/customize_admin.rst

```diff
@@ -0,0 +1,65 @@
+Customize admin
+================
+
+Sonata page bundle there are few admins that can be customize using `Admin Extension`_
+As example if you want to add a new field in ``configureListFields``, You can do like this
+
+.. code-block:: php
```

Comment:
Hey @VincentLanglet  any ida why lint is complaining about this line?

it's the same of : https://raw.githubusercontent.com/sonata-project/SonataAdminBundle/4.x/docs/reference/form_types.rst

Reply:
Dont know a lot about this linter, but the message seems explicit.

Reply:
I don't know why, But it works with `::`!